task/aontimer-demo/src/main.rs

```diff
@@ -0,0 +1,83 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#![no_std]
+#![no_main]
+
+use drv_ext_int_ctrl_api::*;
+use drv_ot_aontimer::*;
+use userlib::*;
+
+task_slot!(INT_CONTROLLER, ext_int_ctrl);
+
+task_config::task_config! {
+    clock_frequency_hz: u32,
+    bark_threshold_ticks: u64,
+    bite_threshold_ticks: u64,
+}
+
+//TODO should be replaced by build time constants
+// see jira https://rivosinc.atlassian.net/browse/SW-530
+const AONTIMER_BARK_NOTIFICATION: u32 = 0x1;
+const AONTIMER_TIME_NOTIFICATION: u32 = 0x2;
+
+fn sleep_and_listen(timer: &AonTimer, int_ctrl: &ExtIntCtrl) {
+    sys_log!("Sleeping...");
+    let alarm = sys_get_timer().now;
+    sys_set_timer(Some(alarm + 1000), AONTIMER_TIME_NOTIFICATION);
+    loop {
+        let result = sys_recv_closed(
+            &mut [],
+            AONTIMER_BARK_NOTIFICATION | AONTIMER_TIME_NOTIFICATION,
+            TaskId::KERNEL,
+        )
+        .unwrap();
+        int_ctrl.complete_int(AONTIMER_BARK_NOTIFICATION).unwrap();
+        if result.operation & AONTIMER_BARK_NOTIFICATION != 0x0 {
+            sys_log!("{}", sys_get_timer().now);
```

Comment:
Unless we still need these logs for debugging, I think these should be commented out or removed

Reply:
The whole point of the task is a demo, so I think it is fine to leave the prints in.